wordpress-mobile/WordPress-Android
# [Bloganuary] Dashboard Nudge card

Part of #19663 

This only contains the UI part of the Bloganuary Nudge Card and basic logic to show it on the dashboard, the interactions are still TODOs and will be done in different PRs.

Light Mode | Dark Mode
--- | ---
![bloganuary-nudge-light-mode](https://github.com/wordpress-mobile/WordPress-Android/assets/5091503/0639afe0-c665-4515-a2f8-66ac43038fca) | ![bloganuary-nudge-dark-mode](https://github.com/wordpress-mobile/WordPress-Android/assets/5091503/2da2aa66-cf06-4fb3-9110-7f1a8985aa18)


-----

## To Test:

1. Install and log into Jetpack (with a site that can see Blogging Prompts)
2. Go to Debug settings
3. Confirm the `bloganuary_dashboard_nudge` FF is ON (should be by default)
4. Go to My Site Dashboard
5. **Verify** the "Bloganuary is coming" card is shown (the buttons/interactions still don't work)
6. Go to Debug settings
7. Disable the `bloganuary_dashboard_nudge` FF
8. Go to My Site Dashboard
9. **Verify** the Bloganuary card is not shown

-----

## Regression Notes

1. Potential unintended areas of impact

    - Other cards UI/behavior affected.

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Manual testing

10. What automated tests I added (or what prevented me from doing so)

    - Unit tests for Bloganuary card builder
    - Unit tests for Bloganuary ViewModel slice
    - Updated unit tests for existing modified classes

-----

## PR Submission Checklist:


-----

## UI Changes Testing Checklist:
